sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClient.java

```diff
@@ -55,6 +55,15 @@
             .setScheduler(connectionOptions.getScheduler());
     }
 
+    /**
+     * Gets the Event Hub name this client interacts with.
+     *
+     * @return The Event Hub name this client interacts with.
+     */
+    public String getEventHubName() {
```

Comment:
Should this be public?

Reply:
See: https://github.com/Azure/azure-sdk-for-java/pull/5474#discussion_r327369142